src/main/fc/cli.c

```diff
@@ -2964,6 +2964,7 @@ static void cliStatus(char *cmdline)
     UNUSED(cmdline);
 
     cliPrintLinef("System Uptime: %d seconds", millis() / 1000);
+    cliPrintLinef("Virtual Date/time: %d-%02d-%02d %02d:%02d:%02d", txInfo.year, txInfo.month, txInfo.day, txInfo.hour, txInfo.minute, txInfo.second);
```

Comment:
Not sure this should be called 'virtual', it is the actual date / time. Also, we should probably suppress printing the info if `txInfo` has never been set (users without SmartPort or the lua script).

Reply:
Agreed. There is nothing virtual about this datetime.